queue_services/entity-emailer/src/entity_emailer/email_processors/ar_overdue_stage_1_notification.py

```diff
@@ -0,0 +1,100 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Email processing rules and actions for stage 1 overdue ARs notifications."""
+from __future__ import annotations
+
+from pathlib import Path
+
+from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Template
+from legal_api.models import Business, Furnishing
+
+from entity_emailer.email_processors import (
+    get_entity_dashboard_url,
+    get_jurisdictions,
+    get_recipient_from_auth,
+    substitute_template_parts,
+)
+
+
+def process(email_info: dict, token: str) -> dict:  # pylint: disable=too-many-locals, , too-many-branches
+    """Build the email for Involuntary dissolution notification."""
+    logger.debug('ar_overdue_stage_1_notification: %s', email_info)
+    # get business
+    furnishing_id = email_info['data']['furnishing']['furnishingId']
+    furnishing = Furnishing.find_by_id(furnishing_id)
+    business = get_business_by_furnishing_id(furnishing_id)
```

Comment:
please create a separate PR and add a relationship to the furnishing model so you can just access the business using `furnishing.business`.